jupyter-book/trajectories/pseudotemporal.ipynb

```diff
@@ -15,53 +15,95 @@
    ]
```

Comment:
<p>Same comment as for RNA velocity. Please make the first cell the title cell (# Trajectory inference).</p><p> </p><p>"However, as cells are destroyed when sequenced, it is impossible to track their development and, for example,  gene expression profile over time. "</p><p>Technically not impossible -&gt; https://www.nature.com/articles/s41586-022-05046-9</p><p> </p><p>It's just really not feasible at scale yet :) And I'm not even convinced whether this is a good idea...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Sorry, I'm confused. The first cell is the title cell. Or do you want it renamed to <code>Trajectory inference</code> ? I thought you'd prefer <code>Pseudotemporal ordering</code> .</p>

Reply:
Oh, good. For some reason the title cell doesn't/didn't show up for me. Keep it as is, no worries.